test_stack.py

```diff
@@ -0,0 +1,44 @@
+import io, pytest, sys
+
+from stack import Stack, LinkedList
+
+
+def test_stack():
+    s = Stack([3, 6, 2])
+
+    captureOutput = io.StringIO()
+    sys.stdout = captureOutput
+    print(s.storage)
+    sys.stdout = sys.__stdout__
```

Comment:
What is the purpose of this test?

Reply:
i wasn't sure how to test the stack set up. i originally had the use of .head, but that would be testing the linkedlist and not the stack so this was what i came up with.

Reply:
This is also testing the linked list.  You should use push pop and __len__ only for your tests. And this should be sufficient to know if the Stack is working correctly.

Reply:
okay